Apps/W1/Shopify/app/src/GraphQL/Codeunits/ShpfyGQLCompany.Codeunit.al

```diff
@@ -13,7 +13,7 @@ codeunit 30302 "Shpfy GQL Company" implements "Shpfy IGraphQL"
     /// <returns>Return value of type Text.</returns>
     internal procedure GetGraphQL(): Text
     begin
-        exit('{"query":"{company(id: \"gid://shopify/Company/{{CompanyId}}\") {name id note createdAt updatedAt mainContact { id customer { id firstName lastName email phone}} locations(first:1, sortKey: CREATED_AT ) {edges { node { id name billingAddress {address1 address2 city countryCode phone province zip zoneCode} taxRegistrationId}}}}}"}');
+        exit('{"query":"{company(id: \"gid://shopify/Company/{{CompanyId}}\") {name id note createdAt updatedAt mainContact { id customer { id firstName lastName email phone}} locations(first:20, sortKey: CREATED_AT ) {edges { node { id name billingAddress {address1 address2 city countryCode phone province zip zoneCode} buyerExperienceConfiguration {paymentTermsTemplate {id}} taxRegistrationId}}}}}"}');
```

Comment:
Is 20 okay with MS? I know sometimes they're okay with hardcoded "limits", but othertimes they implement the looping logic, so just curious.

Reply:
Probably we gonna ask Andrei if it's okay, because in Shopify Connector it seams that it's common practice to hardcode limits:
![image](https://github.com/user-attachments/assets/937c1754-e48b-4b0a-834f-675d677876bf)

Reply:
Yeah, but a lot of these queries also specify the "after:" parameter, so they can receive the next X amount of records, and are used in a loop. 

But I agree it's best to ask Andrei, maybe with Locations it's enough to have a certain batch.